components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -59,6 +59,25 @@ spec:
         securityContext:
         {{- toYaml .Values.pump.containerSecurityContext | nindent 12 }}
         {{- end }}
+
+        {{- if .Values.pump.livenessProbe }}
+        livenessProbe:
+          httpGet:
+            scheme: "HTTP"
+            path: /{{ .Values.pump.service.healthEndpointName | default "health" }}
```

Comment:
If you set `health` in values.yaml, it will be used as default. You don't need to provide default value here again.
You can set it to `/health` to avoid setting `/` here